### ISSUE TITLE
New license request: LAMatplotlib-1.3 [SPDX-Online-Tools]

### ISSUE THREAD

**Issue by OliverFendt** [spdx/license-list-XML#2505]
**1.** License Name: License agreement for matplotlib v1.3.0 and later
**2.** Short identifier: LAMatplotlib-1.3
**3.** License Author or steward: Matplotlib Development Team
**4.** Comments: The Matplotlib license is a unique license and I did not find it in the list of licenses.
It is an OSD conformant license.
**5.** License Request Url: http://tools.spdx.org/app/license_requests/375
**6.** URL(s): https://matplotlib.org/stable/project/license.html
**7.** OSI Status: Not Submitted
**8.** Example Projects: https://github.com/matplotlib/matplotlib

**Comment by richardfontana** [comment]
I agree with @OliverFendt that this license is OSD-conformant. Also, this license is found in major Linux distros such as Fedora (though an issue to review and add it to fedora-license-data has not yet been created). See e.g. https://src.fedoraproject.org/rpms/python-matplotlib

**Comment by swinslow** [comment]
Looking at https://matplotlib.org/stable/project/license.html, I think this may be [PSF-2.0](https://spdx.org/licenses/PSF-2.0.html), with replaceable text taken into account?

**Comment by richardfontana** [comment]
By Jove, I think you're right!

**Comment by OliverFendt** [comment]
It is related to the PYTHON SOFTWARE FOUNDATION LICENSE VERSION 2 but below the license for versions 1.3 or later of Matplotlib there is still the license of prior versions. 
Additionally I think it is wrong to determine that MatPlotlib is licensed under PYTHON SOFTWARE FOUNDATION LICENSE VERSION 2. The license steward of PSF-2.0 is the Python Software Foundation, for Matplotlib it is not the Python Software Foundation, it is the Matplotlib Development Team
In my opinion these two topics qualify for an extra license

**Comment by swinslow** [comment]
@OliverFendt I believe the license for earlier versions of matplotlib is also a match for PSF-2.0, just with JDH's initials instead etc.

Saying that the license identifier corresponding to this license text is PSF-2.0 doesn't imply anything about who manages a project or repository that uses the license. But where a license text is substantively the same as a license that is already on the license list, we wouldn't add another separate identifier if it is substantively identical. This is described for instance in the guideline for [replaceable text](https://spdx.github.io/spdx-spec/v2.3/license-matching-guidelines-and-templates/#b34-guideline-replaceable-text) in the SPDX Matching Guidelines.

**Comment by karsten-klein** [comment]
**{metæffekt} Universe**
canonical name: [matplotlib License Agreement 1.3.0](https://github.com/org-metaeffekt/metaeffekt-universe/blob/main/src/main/resources/ae-universe/[m]/README.md#user-content-matplotlib-License-Agreement-1.3.0:~:text=matplotlib%20License%20Agreement%201.3.0)
short name: matplotlib-1.3.0
markers: General Terms Matches Marker
category: matplotlib
OSI status: _none_

**ScanCode**
matched id: [python](https://github.com/nexB/scancode-toolkit/blob/develop/src/licensedcode/data/licenses/python.LICENSE)
matched id: [unknown-license-reference](https://github.com/nexB/scancode-toolkit/blob/develop/src/licensedcode/data/licenses/unknown-license-reference.LICENSE)

**Comment**
+1 to add as separate SPDX license. Proposing license id to be matplotlib-1.3.0. Rationale: no need to encode the type in the identifier (LA). Use full version if provided (1.3.0 instead of 1.3).

**Comment by jlovejoy** [comment]
finally carved out some time to look a bit closer here:

My understanding is that the license text in the gray box at https://matplotlib.org/stable/project/license.html includes two licenses: one for matplotlib versions 1.3.0 and later and one for matplotlib versions prior to 1.3.0 

In other words, these two licenses don't apply together, but for different versions of the software.

Assuming that is right, I used the SPDX-diff tool to compare each separately to existing licenses on the SPDX License List. Here's what I found:
- matplotlib versions 1.3.0 and later: this is indeed a match to `PSF-2.0` - the only difference being in the name, e.g., "matplotlib" instead of "Python" and "MDT" instead of "PSF", which is not a substantive difference as per the matching guidelines as noted above by @swinslow 
- matplotlib versions prior to 1.3.0: similarly, this also matches `PSF-2.0` with similar name differences of "matplotlib" instead of "Python" but with "JDH" instead of "PSF"


@OliverFendt - it's been a bit, hope you are well! 
You might recall in the early days of SPDX, we started developing the matching guidelines and one of the motivations was all the "vanity" BSD licenses that were the same license, but with only a different name in clause 3. This is essentially the same situation: Python created a license that uses their name. Apparently matplotlib decided to use the same license, but since the PSF didn't "templatize" the license, matplotlib just replaced the names. It is still the same license in terms of the substantive text, obligations, etc.  This is not an uncommon scenario and we handle it be making the name able to be replace with other names. We just haven't seen it for `PSF-2.0` until now.
